Dockerfile

```diff
@@ -31,7 +31,8 @@ COPY . $APP_HOME
 
 RUN ENV=docker ./manage.py collectstatic --noinput \
 && chown -R document-merge-service:root /var/www/static \
-&& chmod 770 -R /var/www/static
+&& chmod 770 -R /var/www/static \
+&& cp /bin/sleep /bin/dms_test_sleep
```

Comment:
Why do you need a separate binary for that?

Reply:
So we can kill it by name, otherwise we might kill some other sleep.

Reply:
ah I see. Since this is only used for tests, it would be neat to handle this in a fixture instead.
Something along the lines of:

```python
@pytest.fixture
def create_dms_test_sleep():
    create_copy()
    yield
    delete_copy()
```

But maybe that's just over-engineering :shrug:

Reply:
> Something along the lines of

Let's do that but, I push this to production first. It is kind of urgent.